tests/integration_tests/full_install.sh

```diff
@@ -34,7 +34,8 @@ fi
 . $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh test_data test_data_clavrx $test_exit $install_script
 . $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh test_data test_data_gpm $test_exit $install_script
 . $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh test_data test_data_sar $test_exit $install_script
-. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh test_data test_data_scat $test_exit $install_script
+. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh test_data test_data_scat_1.11.2 $test_exit $install_script
+. $GEOIPS_PACKAGES_DIR/geoips/setup/check_system_requirements.sh test_data test_data_scat_1.11.3 $test_exit $install_script
```

Comment:
Does this need 1.11.2 and 1.11.3 ?

Reply:
Not necessarily. I was just using all of the data you posted in the ``test_data_urls`` repo alongside other data that was needed (test_data_noaa_aws, ...). Should I remove 1.11.2?